src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.AssemblyInfoInputs.cache

```diff
@@ -0,0 +1 @@
+7ab900b7ee62734caa4ef82dbf5346e3f1e8fff9c3ae99a5270edef9b1195374
```

Comment:
The obj directory can be added to .gitignore.

Reply:
Done